DuckDuckGo/NetworkProtectionStatusViewModel.swift

```diff
@@ -86,20 +87,56 @@ final class NetworkProtectionStatusViewModel: ObservableObject {
 
     @MainActor
     private func enableNetP() async {
-        do {
-            try await tunnelController.setState(to: true)
-        } catch {
-            statusMessage = error.localizedDescription
-            isNetPEnabled = false
-        }
+        await tunnelController.start()
     }
 
     @MainActor
     private func disableNetP() async {
-        do {
-            try await tunnelController.setState(to: false)
-        } catch {
-            statusMessage = error.localizedDescription
+        await tunnelController.stop()
+    }
+
+    private class func titleText(connected isConnected: Bool) -> String {
+        isConnected ? UserText.netPStatusHeaderTitleOn : UserText.netPStatusHeaderTitleOff
+    }
+
+    private class func statusImageID(connected isConnected: Bool) -> String {
+        isConnected ? "VPN" : "VPNDisabled"
+    }
+}
+
+private extension ConnectionStatus {
+    var message: String {
+        switch self {
+        case .notConfigured:
+            return "Not Configured"
+        case .disconnected:
+            return "Disconnected"
```

Comment:
Adding to my comment above, in macOS the text for `.disconnected` is "Not connected".  I also checked the designs and it seems it's the same for iOS.